Installed dependencies set up schema

joieariel opened the pull request.

mengditang commented on prisma/schema.prisma:
```diff
@@ -0,0 +1,63 @@
+  id                  Int      @id @default(autoincrement())
+  rating              Int
+  comment             String
+  image_url           String
```
Do we want to allow multiple pictures in one review?

mengditang commented on prisma/schema.prisma:
```diff
@@ -0,0 +1,63 @@
+
+model User {
+  id         Int      @id @default(autoincrement())
+  first_name String
+  last_name  String
+  username   String   @unique
+  email      String   @unique
+  status     String
+  birthdate  DateTime
+  zip_code   String
+  city       String
+  state      String
+  createdAt  DateTime @default(now())
```
Please pick a naming convention for field names and use it constantly - either camel case or underscore.

mengditang commented on prisma/schema.prisma:
```diff
@@ -0,0 +1,63 @@
+  communityCenter     CommunityCenter @relation(fields: [communityCenter_id], references: [id])
+}
+
+model CommunityCenter {
+  id           Int      @id @default(autoincrement())
+  name         String
+  address      String
+  email        String
+  zip_code     String
+  phone_number String
+  image_url    String
+  description  String
+  latitude     Float
+  longitude    Float
+  createdAt    DateTime @default(now())
+
+  reviews      Review[]
+}
```
You may want to think of how to store hours for community centers as we want to be able to only look at centers that are 'open now'.

mengditang commented:
Also a description of the major changes made in this PR would be really helpful for reviewers to know what to look at. Something to include in the description if applicable: What, Why, How, Testing, Screenshots, Additional Notes.

joieariel commented:
Changed field names to snake case and model names to camel case. Added a ReviewImage table to the schema to allow for multiple images per review. This approach is more flexible than just creating an images field with an array of review image urls, which was my initial idea. Created a new table for the hours of the community center.
